fuzz/asn1.c

```diff
@@ -292,8 +293,7 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
 {
     int n;
 
-
-    for (n = 0; item_type[n] != NULL; ++n) {
+    for (n = 0; n < OSSL_NELEM(item_type); ++n) {
```

Comment:
Not sure why this needed changing - the NULL check was ok.

Reply:
I would probably revert these NULL check changes.

Reply:
I prefer something more straightforward as a size deduced by compiler than expecting a NULL ending item.